hyperscale/pg-13/Dockerfile

```diff
@@ -0,0 +1,69 @@
+# This file is auto generated from it's template,
+# see citusdata/tools/packaging_automation/templates/docker/latest/latest.tmpl.dockerfile.
```

Comment:
Is this comment correct? Is PG13 Dockerfile generated from latest.tmpl.dockerfile ?

I do not think that this comment is valid now.

Can we update all the comments and provide some explanations on how to update these dockerfiles ?

Reply:
Removed the template comments since these docker files are not template-generated